backend/account/authentication_controller.py

```diff
@@ -177,10 +177,13 @@ def set_user_organization(self, request: Request, organization_id: str) -> Respo
                         f"{ErrorMessage.ORGANIZATION_EXIST}, \
                             {ErrorMessage.DUPLICATE_API}"
                     )
-            self.create_tenant_user(organization=organization, user=user)
+            tenant_user = self.create_tenant_user(organization=organization, user=user)
 
             if new_organization:
                 try:
+                    tenant_user.is_login_onboarding_msg = False
```

Comment:
Is it better to use a single json instead of multiple columns for these type statuses?

Reply:
@nehabagdia Should this changes be added now ?